net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/mods-enabled-ldap

```diff
@@ -78,7 +78,7 @@ ldap {
         group {
                 base_dn = "${..base_dn}"
 {%     if helpers.exists('OPNsense.freeradius.ldap.group_filter') and OPNsense.freeradius.ldap.group_filter != '' %}
-                filter = '(objectClass=posixGroup)'
+                filter = "{{ OPNsense.freeradius.ldap.group_filter }}"
```

Comment:
Why not single quotes?

Reply:
I would go for single quotes everywhere but it seems to be handled a bit inconsistently. I'm more than happy to update.